docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -6,6 +6,7 @@ Excel add-ins run across multiple versions of Office, including Office 2016 or l
 
 |  Requirement set  |  Office 365 for Windows\*  |  Office 365 for iPad  |  Office 365 for Mac  | Office Online  |  Office Online Server  |
```

Comment:
Just noticed this (seemingly errant asterisk after the **Office 365 for Windows** table heading.  Can you please remove it?